website/docs/enterprise/entsupport.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Support
+---
+
+Enterprise authentik provides dedicated support, with a Support center where you can open a request and view the progress and communications for your current requests.
+
+### Managing tickets and requests
+
+To access the Requests page, where you can open a request and view current requests, go to the Customer Portal and then click **Support** in the top menu.
+
+You can also bookmark the direct link to your Requests page, using the following URL:
+
+> <https://authentiksecurityinc.zendesk.com/hc/en-us/requests>.
```

Comment:
```suggestion
> <https://customers.goauthentik.io/l/support>.
```

This allows us to easier change the link in the future

Reply:
fixed in recent commit, thanks!